lib/id3c/cli/command/geocode.py

```diff
@@ -248,53 +245,8 @@ def standardize_address(address_series: pd.Series,
     return std_address
 
 
-@contextmanager
-def pickled_cache(filename: str = None) -> TTLCache:
-    """
-    Context manager for reading/writing a :class:`TTLCache` from/to the given
-    *filename*.
-
-    If *filename* exists, it is unpickled and the :class:`TTLCache` object is
-    returned.  If *filename* does not exist, an empty cache will be returned.
-    In either case, the cache object will be written back to the given
-    *filename* upon exiting the ``with`` block.
-
-    If no *filename* is provided, a transient, in-memory cache is returned
-    instead.
-
-    >>> with pickled_cache("/tmp/id3c-geocoding.cache") as cache:
-    ...     cache["key1"] = "value1"
-
-    >>> with pickled_cache("/tmp/id3c-geocoding.cache") as cache:
-    ...     print(cache["key1"])
-    value1
-    """
-    empty_cache = TTLCache(maxsize = CACHE_SIZE, ttl = CACHE_TTL)
-
-    if filename:
-        LOG.info(f"Loading cache from «{filename}»")
-        try:
-            with open(filename, "rb") as file:
-                cache = pickle.load(file)
-        except FileNotFoundError:
-            LOG.warning(f"Cache file «{filename}» does not exist; starting with empty cache.")
-            cache = empty_cache
-        else:
-            assert isinstance(cache, TTLCache), \
-                f"Cache file contains a {cache!r}, not a TTLCache"
-    else:
-        LOG.warning(f"No cache file provided; using transient, in-memory cache.")
-        cache = empty_cache
-
-    try:
-        yield cache
-    finally:
-        if filename:
-            with open(filename, "wb") as file:
-                pickle.dump(cache, file)
-
 
-def get_response_from_cache_or_geocoding(address: dict,
+def get_geocoded_address(address: dict,
```

Comment:
`get_response_from_cache_or_geocoding` looks to be imported by various ETLs in id3c-customizations https://github.com/search?q=org%3Aseattleflu+get_response_from_cache_or_geocoding&type=code I think this rename would be a breaking change without a coordinated update in id3c-customizations.

Reply:
Scratch that! Just saw the id3c-customizations PR here: https://github.com/seattleflu/id3c-customizations/pull/160

Reply:
Sorry, I should have explicitly pointed from this PR to the id3c-customizations one instead of pointing in only one direction.